app/services/participants/transfer/npq.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+module Participants
```

Comment:
I think this should probably be under `services` directly (see review guidance), thoughts?

Reply:
I'd put it under `NPQ` namespace

Reply:
I've moved to under `transfers/` as per Peter's comment as we might have other transfers to add